packages/api/src/utils/client/httpClient.ts

```diff
@@ -198,8 +198,7 @@ export class HttpClient implements IHttpClient {
               this.logger?.debug("Requesting fallback URL", {routeId, baseUrl: printableUrl, score: this.urlsScore[i]});
             }
 
-            // eslint-disable-next-line @typescript-eslint/naming-convention
```

Comment:
naming convention no longer covered?

Reply:
It is disabled for certain files to fix these later to have consistent naming convention across packages. 

https://github.com/ChainSafe/lodestar/blob/nh%2Fbiomejs/biome.jsonc#L297-L298

Reply:
Not ideal that we disable this for the whole file if there are just 1-2 lines affected